### PR TITLE
Add Optimization theory section

### PR DESCRIPTION
**Pull request by tapyu**

### CODE REVIEW

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Thanks for your PR @tapyu! Could you keep only the free and open resources?

File: README.md [hunk 1]
```diff
@@ -67,6 +68,20 @@
 
 - [Signal Processign with GW150914 Open Data](https://losc.ligo.org/s/events/GW150914/GW150914_tutorial.html) - Typical signal processing tasks with the LIGO GW150914 Open Data.
 
+## Optimization
+
+### Learning
+
+- [Convex optimization Solution manual](https://egrcc.github.io/docs/math/cvxbook-solutions.pdf) - Boyd, S.P. and Vandenberghe, L., 2004. Convex optimization. Cambridge university press.
+- [Additional exercises](https://egrcc.github.io/docs/math/cvxbook-solutions.pdf) - Boyd, S.P. and Vandenberghe, L., 2004. Convex optimization. Cambridge university press.
+- [EE364A - Convex Optimization I](https://see.stanford.edu/Course/EE364A) and [Material](https://web.stanford.edu/class/ee364a/) - Stanford Engineering Everywhere - Convex Optimization I course.
```
Comment (tapyu) [inline README.md]: Hi! the course platform "Stanford Engineering Everywhere" is free (in the "free beer" sense") and open. There is not charge to access this course, at all.

**Comment by tapyu** [comment]
I am not sure if it is necessary, but I just added the free icon to make it clear that there is no charge in the "Stanford Engineering Everywhere" platform.

**Comment by astrojuanlu** [comment]
Oh, thanks! Please solve the merge conflicts and I'll merge

**Comment by tapyu** [comment]
Done.

**Pull request closed without merging**